tests/ci/profile_handler.go

```diff
@@ -355,7 +369,6 @@ func GenerateClusterCreationArgsByProfile(token string, profile *Profile) (clust
 			}
 			clusterArgs.MachineCIDR = vpcOutput.VPCCIDR
 			if profile.AdditionalSGNumber != 0 {
-				var sgIDs []string
 				// Prepare profile.AdditionalSGNumber+5 security groups for negative testing
 				sgIDs, err = PrepareAdditionalSecurityGroups(profile.Region, vpcOutput.VPCID, profile.AdditionalSGNumber+5)
```

Comment:
Let's not use additional security groups in case the cluster is AdditionalSGNumber not enabled but proxy enabled. It's a good choice to put the SG in proxy instance start up in your previous code. Just notice that make CIDR value parameter.